### PR TITLE
Apply fix to controller macros suggested by parndt

### PR DESCRIPTION
**Pull request by CJBrew**
- to remove deprecation warnings in rspec run

Sending pull request in order to testing this in the Travis CI build -- 

it may fix https://github.com/refinery/refinerycms/issues/2927 -- my tests seem to pass.

### CODE REVIEW

**Comment by CJBrew** [comment]
Needs more work to fix tests....

**Comment by CJBrew** [comment]
This seems to do the job... :)

**Comment by CJBrew** [comment]
Is this an intermittent test failure? I was seeing it on my other branch (not related).

```
Failures:
  1) Pages delete existing page part deletes page parts
     Failure/Error: expect(page).to have_no_content("Second Part")
       expected #has_no_content?("Second Part") to return true, got false
     # ./pages/spec/features/refinery/admin/pages_spec.rb:675:in `block (4 levels) in <module:Admin>'
     # ./pages/spec/features/refinery/admin/pages_spec.rb:673:in `block (3 levels) in <module:Admin>'
     # ./core/spec/support/database_cleaner.rb:18:in `block (2 levels) in <top (required)>'
     # -e:1:in `<main>'
Finished in 44.86 seconds (files took 0.69144 seconds to load)
233 examples, 1 failure
Failed examples:
rspec ./pages/spec/features/refinery/admin/pages_spec.rb:656 # Pages delete existing page part deletes page parts
```

**Comment by parndt** [comment]
@CJBrew yes, it's our newest intermittent failure

**Comment by parndt** [comment]
Thank you so much!

File: core/spec/controllers/refinery/admin/dummy_controller_spec.rb [hunk 1]
```diff
@@ -2,6 +2,14 @@
 
 module Refinery
   module Admin
+    class Engine < ::Rails::Engine
+      isolate_namespace Refinery::Admin
```
Comment (simi) [inline core/spec/controllers/refinery/admin/dummy_controller_spec.rb]: This is insane :(

`isolate_namespace` is meant to be used in engine as we do [already](https://github.com/refinery/refinerycms/blob/1a8278498a5d9eaa7a0db4946d2055a9ee83b634/core/lib/refinery/core/engine.rb#L6).

I'm working on proper fix for this.

**Pull request merged**